### PR TITLE
10 new Diona "hair" styles

### PR DESCRIPTION
**Pull request by ExusA**
## What Does This PR Do
Adds 10 new hairstyles for dionea.

## Why It's Good For The Game
Hairstyle is the only customisation option avaliable to diona players. There needs to be lots of choices for characters to feel more unique! I felt there weren't enough unique-feeling hair-styles avaliable, so I made some more. I'm also an amateur spriter, so I hope they look okay!

## Images of changes
This is going to be long...

sunflower
-Sunflower

marigold
-Marigold

lilies
-Lilies

frost
-Frost

daisies
-Daisies

bigrose
-Big Rose

poppies
-Poppies

bigleaf
-Big Leaves

field mushrooms
-Field Mushrooms

hydrangea
-Hydrangea

## Changelog
:cl:
imageadd: Added 10 diona hairstyles.

### CODE REVIEW

**Comment by ExusA** [comment]
BEFORE I FORGET: The original "laurel" hairstyle had a few odd white pixels hanging out on the border of it, so I cleaned them up, and that's why the diff includes it.

**Comment by PacifistDalek** [comment]
I'm a diona main and I love all of these

**Comment by Darkmight9** [comment]
Oh this is very lovely! ❤️ 
Maybe for a future PR you could make 'long hair' options such as vines(both bare green, covered in flowers, maybe tied into a ponytail, etc), a really big leaf that hangs off the back, possibly some others (still waking up)

**Comment by Spacemanspark** [comment]
I wish nymphs could randomly spawn with a little flower on their head like this too

**Review by Kyep — APPROVED** [review]

File: code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm [hunk 1]
```diff
@@ -57,3 +57,48 @@
...
+	icon_state = "frost"
+
+/datum/sprite_accessory/hair/diona/diona_largerose
+	name = "Large Rose"
+	icon_state = "largerose"
+
+/datum/sprite_accessory/hair/diona/diona_lily
+	name = "Lilies"
+	icon_state = "lily"
+	glasses_over = 0
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm:74]:
```suggestion
	glasses_over = FALSE
```

File: code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm [hunk 2]
```diff
@@ -57,3 +57,48 @@
...
+
+/datum/sprite_accessory/hair/diona/diona_sunflower
+	name = "Sunflower"
+	icon_state = "sunflower"
+	glasses_over = 0
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm:79]:
```suggestion
	glasses_over = FALSE
```

File: code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm [hunk 3]
```diff
@@ -57,3 +57,48 @@
...
+	icon_state = "marigold"
+
+/datum/sprite_accessory/hair/diona/diona_hydrangea
+	name = "Hydrangea"
+	icon_state = "hydrangea"
+
+/datum/sprite_accessory/hair/diona/diona_bigleaf
+	name = "Big Leaf"
+	icon_state = "bigleaf"
+	glasses_over = 0
```
Comment (AffectedArc07) [inline code/modules/mob/new_player/sprite_accessories/diona/diona_hair.dm:104]:
```suggestion
	glasses_over = FALSE
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

**Comment by variableundefined** [comment]
Pretty and unique

**Pull request merged**